<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myStatisticPic</class>
 <widget class="QWidget" name="myStatisticPic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myStatisticPic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QChartView" name="pieToday" native="true"/>
     </item>
     <item>
      <widget class="QChartView" name="pieWhole" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QChartView" name="lineChart" native="true"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
